### ISSUE TITLE
fatal error due to jvm crash

### ISSUE THREAD

**Issue by sureshgurram123** [adoptium/adoptium-support#1091]
### Please provide a brief summary of the bug

[hs_err_pid1680.log](https://github.com/adoptium/adoptium-support/files/15330800/hs_err_pid1680.log)

### Did you test with the latest update version?

- [X] Yes

### Please provide steps to reproduce where possible

We have noticed that metacache utilization reaching out 99% and the jvm crashes after few minutes.

### Expected Results

Jboss service continue to run without any issues

### Actual Results

Due to the jvm crash, the jboss service that is using the java is also getting stopped. 

### What Java Version are you using?

JRE version: OpenJDK Runtime Environment Temurin-11.0.12+7 (11.0.12+7) (build 11.0.12+7)

### What is your operating system and platform?

Windows 2019

**Comment by karianna** [comment]
Please move to 11.0.23 and try again.

**Comment by sureshgurram123** [comment]
Hi Martijn,

Thanks for the assistance. I have updated the jdk to 11.0.23 and the issue has been solved with this.
Is it possible to give more details on the exact with jdk-11.0.12_7 and what's been fixed in 11.0.23 ?

We have many places where we need to update the java ( even though there is no issue but to maintain the consistency), so the explanation could help us to plan the changes needed.

Appreciate your help.

Thanks
Suresh
